Formula/mtm.rb

```diff
@@ -0,0 +1,40 @@
+class Mtm < Formula
+  desc "Micro terminal multiplexer"
+  homepage "https://github.com/deadpixi/mtm"
+  url "https://github.com/deadpixi/mtm/archive/1.2.0.tar.gz"
+  sha256 "68f753f743fcd3d87f269011d4bbd3fed59cfcad710d7c7db98844e3e675c196"
+  license "GPL-3.0-or-later"
+
+  head do
+    url "https://github.com/deadpixi/mtm.git"
+
+    uses_from_macos "ncurses"
+  end
+
+  depends_on "ncurses" # requires ncurses >= 6.1
+
+  def install
+    bin.mkpath
+    man1.mkpath
+
+    makefile = build.head? ? "Makefile.darwin" : "Makefile"
```

Comment:
Why this?

Reply:
Because upstream introduced dedicated `Makefile.darwin` after stable got released. Basically, it went from having a single `Makefile` to both `Makefile` & `Makefile.darwin` and it is probably a good idea to use the former if possible?